recipe/0001-Add-more-locations-to-find-the-dri.patch

```diff
@@ -0,0 +1,31 @@
+From d3c233b6e37952f60e705bcd80a3b00fdc40e079 Mon Sep 17 00:00:00 2001
+From: Mark Harfouche <mark.harfouche@gmail.com>
+Date: Sat, 1 Mar 2025 23:19:40 -0500
+Subject: [PATCH] Add more locations to find the dri
+
+---
+ meson.build | 8 +++++++-
+ 1 file changed, 7 insertions(+), 1 deletion(-)
+
+diff --git a/meson.build b/meson.build
+index 9b71bf7..369ac6a 100644
+--- a/meson.build
++++ b/meson.build
+@@ -75,7 +75,13 @@ if driverdir == ''
+   if host_machine.system() == 'windows'
+     driverdir = join_paths(get_option('prefix'), get_option('bindir'))
+   else
+-    driverdir = join_paths(get_option('prefix'), get_option('libdir'), 'dri')
++    driverdir = '@0@/dri:@1@/dri:@2@/dri'.format(
++      join_paths(get_option('prefix'), get_option('libdir')),
++      # This is where the Ubuntu package installs the drivers
++      '/usr/lib/x86_64-linux-gnu',
```

Comment:
this is not correct for aarch and ppc64le

Reply:
do you know what they might be?

Reply:
`$(gcc -print-multiarch | sed -e 's/i.*86/i386/')`

@hmaarrfk

Reply:
i can try tomorrow (or you are free to push) my head hurts now! thanks for your help!

Reply:
I'll carry on for now, and try to reduce the patch to one, if plausible enough. I think sed will definitely work.

Reply:
I think it's better to keep the three patches, because the above command is unfortunately not going to work when cross-compiling.